server/models/articleLikes.js

```diff
@@ -0,0 +1,42 @@
+const getArticleLikesModel = (sequelize, Sequelize) => {
```

Comment:
Rename this model to getReactionsModel. Because we are going to use this for likes/dislikes for Articles and Comments. This table should have the following properties:

- id
- reaction
- sourceId 
- sourceType
- createdAt
- updatedAt
**Explanation:**
**reaction** : could be either true or false,  where true represents a like and false represents a dislike
**sourceId**: could be either an Id of an Article or a Comment
**sourceType**: could be either 1 or 2, where 1 represents an Article and 2 represents a Comment